QuickStart/ChatViewController.swift

```diff
@@ -6,17 +6,41 @@
 //
 
 import SendBirdDesk
+import SendbirdChatSDK
 import SendbirdUIKit
 import UIKit
 
 class ChatViewController: SBUGroupChannelViewController {
     var ticket: SBDSKTicket!
     
+    /// When this flag is set to true, system messages will be hidden from the message list view.
+    static let shouldHideSystemMessages = false
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
         self.useRightBarButtonItem = false
     }
+    
+    override func baseChannelModule(_ listComponent: SBUBaseChannelModule.List, fullMessagesInTableView tableView: UITableView) -> [BaseMessage] {
+        let filtered = self.viewModel?.fullMessageList.filter { Self.isVisible(message: $0) || !Self.shouldHideSystemMessages }
+        return filtered ?? []
+    }
+    
+    static func isVisible(message: BaseMessage) -> Bool {
+        if let message = message as? AdminMessage, let data = Data(base64Encoded: message.data), data.isEmpty == false {
```

Comment:
@mininny please modify to use `String.date(...)`

Reply:
Thanks!